src/transaction/log_manager.c

```diff
@@ -1703,7 +1703,7 @@ log_initialize_passive_tran_server (THREAD_ENTRY * thread_p)
 
   logpb_initialize_logging_statistics ();
 
-  pts_ptr->send_oldest_active_mvccid ();	/* TODO not here acutally, just for testing. will be removed soon */
+  pts_ptr->start_oldest_active_mvccid_sender ();
```

Comment:
Let's try to start it sooner, before starting the replicator.
Although in practice it will not make a difference since the message is fire&forget anyway.
Or, better, move the starting of the replicator after this line.

Reply:
I have moved it before starting the replicator. It seems better in that other daemons start there, and it is right after the PTS log analysis which allows the mvccid-sender to start.